### PR TITLE
Fixed external links akka-protobuf and akka-remote

### PR DESCRIPTION
**Pull request by purvaagrawal25**
Fixed External Links (akka-protobuf and akka-remote)

References #30697

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @purvaagrawal25,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[https://www.lightbend.com/contribute/cla](https://www.lightbend.com/contribute/cla)

**Comment by purvaagrawal25** [comment]
I have signed the CLA.

File: akka-protobuf/src/main/java/akka/protobuf/ByteString.java [hunk 1]
```diff
@@ -373,7 +373,7 @@ private static ByteString readChunk(InputStream in, final int chunkSize)
    * Concatenate the given {@code ByteString} to this one. Short concatenations,
    * of total size smaller than {@link ByteString#CONCATENATE_BY_COPY_SIZE}, are
    * produced by copying the underlying bytes (as per Rope.java, <a
-   * href="http://www.cs.ubc.ca/local/reading/proceedings/spe91-95/spe/vol25/issue12/spe986.pdf">
+   * href="https://www.cs.ubc.ca/local/reading/proceedings/spe91-95/spe/vol25/issue12/spe986.pdf">
```
Comment (raboof) [inline akka-protobuf/src/main/java/akka/protobuf/ByteString.java:376]: This file is no longer found - I guess that makes 'https' as good as 'http', but perhaps you could find a working link?

File: akka-remote/src/main/scala/akka/remote/PhiAccrualFailureDetector.scala [hunk 2]
```diff
@@ -21,7 +21,7 @@ import akka.util.Helpers.ConfigOps
 
 /**
  * Implementation of 'The Phi Accrual Failure Detector' by Hayashibara et al. as defined in their paper:
- * [http://www.jaist.ac.jp/~defago/files/pdf/IS_RR_2004_010.pdf]
+ * [https://www.jaist.ac.jp/~defago/files/pdf/IS_RR_2004_010.pdf]
```
Comment (raboof) [inline akka-remote/src/main/scala/akka/remote/PhiAccrualFailureDetector.scala:24]: Same for this one

**Review by raboof — APPROVED** [review]
Two of the links seem to be broken, the others are :+1: .

OK to merge I guess, but could you perhaps find up-to-date links for the two broken ones?

File: akka-remote/src/main/scala/akka/remote/PhiAccrualFailureDetector.scala (hunk 2 above)
Comment (purvaagrawal25) [reply akka-remote/src/main/scala/akka/remote/PhiAccrualFailureDetector.scala]: I was able to find the up-to-date link for this and updated the same.

File: akka-protobuf/src/main/java/akka/protobuf/ByteString.java (hunk 1 above)
Comment (purvaagrawal25) [reply akka-protobuf/src/main/java/akka/protobuf/ByteString.java:376]: Hi @raboof ,
I am not able to find the up-to-date link for this one. I will try this next time.
Thanks

**Comment by raboof** [comment]
Great, thanks!

**Pull request merged**